src/fpm/manifest/preprocess.f90

```diff
@@ -82,27 +82,17 @@ subroutine check(table, error)
 
       character(len=:), allocatable :: name
       type(toml_key), allocatable :: list(:)
-      logical :: suffixes_present, directories_present, macros_present
       integer :: ikey
 
-      suffixes_present = .false.
-      directories_present = .false.
-      macros_present = .false.
-
       call table%get_key(name)
       call table%get_keys(list)
 
       do ikey = 1, size(list)
          select case(list(ikey)%key)
-          case default
-            call syntax_error(error, "Key " // list(ikey)%key // "is not allowed in preprocessor"//name)
-            exit
-          case("suffixes")
-            suffixes_present = .true.
-          case("directories")
-            directories_present = .true.
-          case("macros")
-            macros_present = .true.
+         !> Valid keys.
+         case("suffixes", "directories", "macros", "export-windows-macro")
```

Comment:
```suggestion
         case("suffixes", "directories", "macros")
```
It seems `FPM_IS_WINDOWS` has been removed, so `export-windows-macro` should also be removed?

Reply:
Yes, I overlooked that one